README.md

```diff
@@ -1,76 +1,72 @@
- ## 一个支持多种风格的自定义的密码(验证码)输入框
+# PasswordInputView
+A HMOS library of custom password (verification code) input box that supports multiple styles.
 
- ![image](https://github.com/LambertCoding/PasswordInputView/blob/master/image/password_input_view2.png)
+# Source
+Inspired by [yu1tiao/PasswordInputView](https://github.com/yu1tiao/PasswordInputView)
 
 ## Features
+1) This library can display plaintext or ciphertext, ciphertext supports displaying dots, asterisks, or any characters.
+2) The user can set the color of the border of the input box and password.
+3) The color of the input box changes after providing input.
+4) The user can set the password style as asterisk, plaintext and circle.
+5) The user can set the border style line and box.
+6) The user can set the maximum length of the password.
+7) You can set the interval and rounded corners between the password boxes.
+8) You can configure the radius for the input boxes.
 
-* 仿支付宝微信风格
-* 下划线风格
-* 可显示明文或者密文，密文支持显示圆点，星号，或者任意字符
-* 设置密码框之间的间隔和圆角(间隔为0时圆角只显示最左和最右的圆角)
-* 设置边框和密码的颜色
-* 支持已输入部分单独设置颜色
+<img src="screenshots/passwordinputview.gif" width="500">
```

Comment:
Gif is very blurry and pixelated in some places. Please replace it with better quality gif.

Reply:
Done